### PR TITLE
Remove the tutorial; replace with a quickstart section.

### PR DESCRIPTION
**Pull request by grahamgower**
The tutorial content is out of date with respect to this year's
spec changes and the YAML files there were unfortunately missed
during updates. Also, the tutorial now belongs in the spec repository.

### CODE REVIEW

File: demes/demes.py [hunk 1]
```diff
@@ -2103,7 +2103,8 @@ class Builder:
 
     :ivar dict data: The data dictionary of the graph's current state.
         The objects nested within this dictionary follow Demes' data model,
-        as described in the :ref:`schema <sec_schema>`.
+        as described in the
+        `schema <https://popsim-consortium.github.io/demes-spec-docs/main/spec.html>`_.
```
Comment (jeromekelleher) [inline demes/demes.py:2107]: Better to use intersphinx for this - it's a sphinx site, right, so it has an objects.inv etc.

**Review by jeromekelleher — COMMENTED** [review]
LGTM, but we should use intersphinx.

File: demes/demes.py (hunk 1 above)
Comment (grahamgower) [reply demes/demes.py:2107]: Oh cool, thanks. I've made about 100 tiny mistakes here though... maybe I should just quit and go have a beer instead.

Comment (jeromekelleher) [reply demes/demes.py]: If you have to ask the question, it's definitely time for beer. Enjoy!

**Pull request merged**